sdk/core/core-util/src/sha256.common.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { stringToUint8Array, uint8ArrayToString } from "./bytesEncoding.js";
```

Comment:
also need to use browser version of bytesEncoding.js for react-native.

Reply:
Fixed